test/integration/other/form-entities-version.js

```diff
@@ -64,7 +65,7 @@ describe('Update / migrate entities-version within form', () => {
         .set('Content-Type', 'application/xml')
         .expect(200);
 
-      const { acteeId } = await Forms.getByProjectAndXmlFormId(1, 'updateEntity').then(o => o.get());
+      const { acteeId } = await Forms.getByProjectAndXmlFormId(1, 'updateEntity', Form.WithoutDef).then(o => o.get());
```

Comment:
Just commenting that I appreciate/apologize for you having to go and changes all these things in the tests. Ideally there wouldn't be code like this in the tests, but uh, I guess I did what had to be done for checking things that weren't exposed in the API.

Reply:
No big deal :¬)